content/ADR-145-pick-realm-v2.md

```diff
@@ -59,46 +79,137 @@ https://github.com/decentraland/kernel/blob/main/packages/shared/dao/index.ts#L6
 ```typescript
 export function fetchCatalystStatus() {
 
-  if (currentChecks && result.accepting_users) {
+  if (currentChecks && result.acceptingUsers) {
     // ...
   }
 
   return undefined
 }
 ```
 
-_Configuration_:
+### Catalyst Version
+
+_Description_:
+
+Every Catalyst node provides information about the versions of its Content, Lambdas, BFF, and Comms components. The Kernel reads these version fields and compares them with the minimum required version specified in the rule configuration. As a result, the configuration has the flexibility to specify minimum required versions for one or more services, allowing the Catalyst to be considered as a candidate only if it meets these requirements. This means that the configuration file can specify minimum required versions for X and Xn services, giving the flexibility to specify versions for one or more services as needed.
+
 
-This configuration will be at BFF (Catalyst) level
+_Pseudo Code_:
 
 ```typescript
-export type AcceptingUsersConfig = {
-  config?: 
-          { maxAmountOfUsers: number;
-            maxCpu: number;
-            maxMemory: number;
-          }
+export function catalystVersionLink(configuration: CatalystVersionConfig) {
+  const minimumRequiredVersions = configuration
+
+  Object.keys(minimumRequiredVersions).forEach(([serviceName, serviceRequiredVersion]) => {
+    validCandidates = picked.filter(candidate => meetsVersion(candidate, serviceName, serviceRequiredVersion))
+    // ...
+  })
+
+  return validCandidates
 }
 ```
 
+_Configuration_:
 
-### Catalyst Version Link
+```typescript
+export type CatalystVersionConfig = {
+  config?: 
+          { comms?: string;
+            bff?: string;
+            content?: string;
+            lambdas?: string;
+          }
+}
+```
 
+### Force Catalyst
```

Comment:
The force catalyst will return only one? Does it make sense to return a recommended order instead?